support-frontend/assets/components/productOption/productOption.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
```

Comment:
I think you can roll these together if you want to, as
`import React, { type Node } from 'react';`